charts/consul/values.yaml

```diff
@@ -1859,6 +1859,85 @@ connectInject:
     # Note: This value has no effect if transparent proxy is disabled on the pod.
     defaultOverwriteProbes: true
 
+  # Configures consul-cni plugin for Consul Service mesh services
+  # Using this feature requires Consul 1.13.0-beta+
+  cni:
+    # If true, then all traffic redirection setup with use the consul-cni plugin
+    # @type: boolean
+    enabled: false
+
+    # Namespace to install cni plugin.
+    # @type: string
+    namespace: kube-system
+ 
+    # If multus plugin is enabled or not. If true, the cni installer will generate a consul cni configuration
+    # in a location for the multus program to use. Also, the helm chart will create a custom resource for
+    # NetworkAttachmentDefintion that multis. cniBinDir and cniNetDir must be changed to work with multus.
+    # Please refer to your cloud platform for more informaton on using multus.
+    # @type: boolean
+    multus: false
+
+    # Log level for the installer and plugin. Overrides global.logLevel
+    # @type: string
+    logLevel: null 
+    
+    # Location on the kubernetes node where the CNI plugin is installed. Shoud be the absolute path and start with a '/'
+    # @type: string
+    cniBinDir: "/opt/cni/bin"
+
+    # Location on the kubernetes node of all CNI configuration. Should be the absolute path and start with a '/'
+    # @type: string
+    cniNetDir: "/etc/cni/net.d"
+
+    # The daemonset should in privileged mode on all kubernetes platforms as it needs to interact with the
+    # kubernetes host file system
```

Comment:
If it needs to always be privileged, then why do we allow to set `privileged` to `false`?

Reply:
I was following Istios pattern of allowing for a toggle. I am assuming that someone somewhere would want to toggle it off. Maybe that is premature optimization...

Reply:
I will remove the toggle.

Reply:
The toggle was removed.